src/main/java/appeng/core/sync/packets/PacketValueConfig.java

```diff
@@ -160,6 +161,11 @@ else if( this.Name.equals( "LevelEmitter.Value" ) && c instanceof ContainerLevel
 			final ContainerLevelEmitter lvc = (ContainerLevelEmitter) c;
 			lvc.setLevel( Long.parseLong( this.Value ), player );
 		}
+		else if( this.Name.equals( "FluidLevelEmitter.Value" ) && c instanceof ContainerFluidLevelEmitter )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "FluidLevelEmitter.Value" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1132)